input/cql/HIVB9DTLogic.cql

```diff
@@ -1,5 +1,5 @@
 /*
-@DecisionID: HIV.B9.DT
+@DecisionID: HIV.B9.DT Logic
```

Comment:
"Logic" is the name of the library, etc. but the decision rule is just HIV.B9.DT

```suggestion
@DecisionID: HIV.B9.DT
```

Reply:
This is a different in the SOP for decision rules and indicators. Still we could put:

```
Library: HIV.B9.DT Logic
```

At the start for consistency?